.travis.yml

```
@@ -1,36 +1,20 @@
 language: android
-sudo: false
-jdk: oraclejdk8
-env:
-  matrix:
-    - ANDROID_TARGET=android-18 ANDROID_ABI=armeabi-v7a
-  global:
-    # wait up to 10 minutes for adb to connect to emulator
-    - ADB_INSTALL_TIMEOUT=10
+
+sudo: true
 
 android:
   components:
-    - platform-tools
     - tools
-    - build-tools-23.0.3
-    - android-23
-
-    # Additional components
+    - build-tools-28.0.2
+    - android-27
```

This could be version 28
```suggestion
    - android-28
```